src/interpret.jl

```diff
@@ -282,7 +282,7 @@ end
 function structname(frame, node)
     name = node.args[1]
     if isa(name, GlobalRef)
-        mod = name.module
+        mod = name.mod
```

Comment:
:exploding_head: 
I would almost expect a parsing error for this.

Reply:
Well, it is a valid fieldname (as are all keywords, I think):
```
julia> x = Symbol("module")
:module

julia> @eval struct Foo; $x; end

julia> Foo(2).module
2
```